mysql-test/include/validate_binlog_snapshot_gtid.inc

```diff
@@ -0,0 +1,33 @@
+--let $snapshot_gtid_executed = query_get_value(SHOW STATUS LIKE "Binlog_snapshot_gtid_executed", Value, 1)
```

Comment:
Not sure it this helps but 5.7 and 8.0 have GTID set functions
https://dev.mysql.com/doc/refman/5.7/en/gtid-functions.html

Reply:
Good idea to use GTID_SUBSET() however in this case it will not work because:
1. (Most important) binlog_snapshot_gitd_executed can return 'not-in-consistent-snapshot' value if we are not in transaction with consistent snapshot. This will not work with GTID_SUBSET()
2. We need to check if what we expect is exactly what we get (not the subset)